jenkins/ci.suse.de/pipelines/cloud-gating.Jenkinsfile

```diff
@@ -35,7 +35,7 @@ pipeline {
 
           env.starttime = sh (
             returnStdout: true,
-            script: 'echo $(date +%s)'
+            script: 'date +%s -d "$(wget -q --server-response -O /dev/null http://provo-clouddata.cloud.suse.de/repos/x86_64/SUSE-OpenStack-Cloud-${version}-devel-staging/media.1/build 2>&1 |grep -P ^\ \ Last-Modified:\ .\*|cut -d \  -f 4-)"'
```

Comment:
I agree: very nice.

Just for the sake of nitpicking, this would be a bit shorter (and does not mess with IO redirections):
```
curl -sI  $URL | grep "Last-Modified: " | cut -d' ' -f2-
# and to make sure that the result only outputs one line (header duplicates may occur):
curl -sI  $URL | grep "Last-Modified: " | head -n1 | cut -d' ' -f2-
```

Reply:
Done